src/entry_point.rs

```diff
@@ -85,8 +85,12 @@ pub unsafe extern "C" fn _start(
         ldr r5, =.start   // r5 = address of .start
         cmp r4, r5
         beq .Lstack_init  // Jump to stack initialization if pc was correct
+        movw r0, #8       // LowLevelDebug driver number
+        movw r1, #1       // LowLevelDebug 'print status code' command
+        movw r2, #2       // LowLevelDebug relocation failed status code
+        svc 2             // command() syscall
```

Comment:
Suggestion: Implement this for riscv32 as well?

Reply:
I don't have access to a RISC-V chip or build setup to test with, so that would be very time consuming for me. Alistair already agreed to add a RISC-V implementation, which I appreciate.